src/components/Modals/ConfirmTransaction/index.tsx

```diff
@@ -43,9 +44,10 @@ import { shortenAddress } from 'utils';
 const ConfirmTransaction: React.FC = () => {
   const history = useHistory();
   const { actionType, inputValue, onCloseModal, onDeposit, onWithdraw } = React.useContext(DepositWithdrawalContext);
+  const { totalStaked } = React.useContext(TotalStakedContext);
```

Comment:
Maybe we can also hook-ify this context too?

Reply:
Fixed in 617eb58a838c03f5423bd02ddbf96ea39e5eff43